debian/mariadb-test.install

```diff
@@ -2,6 +2,9 @@ usr/bin/mariadb-client-test
 usr/bin/mariadb-client-test-embedded
 usr/bin/mariadb-test
 usr/bin/mariadb-test-embedded
+usr/lib/*/libmariadb3/plugin/auth_test_plugin.so
```

Comment:
this should be in mariadb-test.links, shouldn't it?

Reply:
No, it is already a symlink, this step just installs it into the symlink file in correct package.